docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,13 +2,174 @@
+investigation.
+
+[role="screenshot"]
+image::siem-ui.png[]
+
+[float]
+[[hosts-ui]]
+=== Hosts
+
+The Hosts view provides summary and detailed information for the hosts you are
+monitoring. You can drag and drop items of interest from the Hosts view to
+Timeline for further investigation.
+
+[role="screenshot"]
+image::hosts-ui.png[]
+
+Interactive widgets let you drill down for deeper insights:
+
+* Hosts
+* Unique IPs
+* User Authentications (success and failures)
+* Uncommon Processes
+* Events
+
+*Hosts Detail* shows you summary information for a particular host, including
+*Host ID, First Seen timestamp, Last Seen timestamp, IP and MAC addresses, OS,
+*versions, machine type, and so forth.
+
+[float]
+[[index-config-hosts]]
+==== Index configuration for hosts events
+
+The default index patterns for hosts data are `auditbeat-*``, `winlogbeat-*``,
+`filebeat-*``, and `packetbeat-*`. You can change the default index patterns in
+{kib} -> Management -> Advanced Settings -> `siem:defaultIndex`.
+
+
+[float]
+[[network-ui]]
+=== Network
+
+The Network view provides summary and detailed information for the network
+traffic you are monitoring. You can drag and drop items of interest from the
+Network view to Timeline for further investigation.
+
+[role="screenshot"]
+image::network-ui.png[]
+
+Interactive widgets let you drill down for deeper insights:
+
+* Top Talkers
+* Top DNS Domains
+* IP Details
+* Domains
+* Users
+* Transport Layer Security certs
+
+[float]
+[[network-index-config]]
+==== Index configuration for network events
+
+The default index patterns for network data are  `filebeat-*``, and
+`packetbeat-*`. You can change the default index patterns in {kib} -> Management
+-> Advanced Settings -> `siem:defaultIndex`.
+
+
+[float]
+[[timelines-ui]]
+=== Timelines
+
+Use timelines to collect evidence and research suspicious activity during an
+investigation. 
+
+You can drag and drop items that warrant further investigation onto the Timeline
+from any place in the {siem-ui}--from Overview, Hosts, Network pages, or even
+from within Timeline itself.
+
+A timeline is responsive and persists as you move through the {siem-app}
+collecting data.
+
+
+[role="screenshot"]
+image::timeline-ui.png[]
+
+Add notes for your own use and to communicate your workflow and findings to
+others. You can share a timeline, or pass it off to another person or team. You
+can link to timelines from a ticketing system.
+
+
+[float]
+[[raw]]
+==== See raw data 
+
+You can click and expand events to see the raw data, including the raw {es} JSON.
+
+[float]
+[[narrow-expand]]
+==== Narrow or expand your query 
+
+You can specify `AND` and `OR` operations with an item's placement in the query
+bar. Horizontal filters are `AND`-ed together. Vertical filters or sets are
+`OR`-ed together. As you hover the item, you can see whether your placement is
+on target to create an `AND` or `OR` statement.
+
+[float]
+[[pivot]]
+==== Pivot your data 
```

Comment:
I think a better way of saying it would be either simply "Pivoting", or if you want to be more specific, perhaps "Pivot on your data", or "Pivot on a data point"

Reply:
@webmat  Hi there. I just found these comments!  After I put up the pre-release placeholders, I stopped working in `master` so that I could keep that placeholder active.  The short version of the story is that none of your changes are in the docs as of this moment.  :-(

With the exception of your comments, everything in this PR has been superseded by #377.
My intention was to close to add this note and close this PR without merging. Instead, I'll leave it open as a reminder that we have outstanding review comments here.

Reply:
And thank you so much for taking time to review and make comments. <3